packages/nimble-components/src/table/index.ts

```diff
@@ -514,6 +559,7 @@ export class Table<
         }
 
         this.observeColumns();
+        this.tableLayoutManager.updateTableViewportMinWidth();
```

Comment:
Why does the table viewport min width need to be updated when the columns change? Looking at the implementation of `updateTableViewportMinWidth`, I'm guessing it has to do with the calculation for the total minimum column width. Is that correct? If so, wouldn't that code need to be run again in the `update` code path due to a column configuration changing?

Reply:
No longer doing this.